platform/linux-generic/include/odp_ipsec_internal.h

```diff
@@ -83,8 +83,22 @@ int _odp_ipsec_status_send(odp_queue_t queue,
 
 #define IPSEC_MAX_SALT_LEN	4    /**< Maximum salt length in bytes */
 
-/* 32 is minimum required by the standard. We do not support more */
-#define IPSEC_ANTIREPLAY_WS	32
+/* The minimum supported AR window size, we do not support less */
+#define IPSEC_ARWIN_SIZE_MIN	32
```

Comment:
I am not sure if it is immediately clear what ARWIN means. I wonder if AR_WIN or just WIN would be better.

Reply:
ACK. Will change ARWIN to AR_WIN for better readability.